src/Symfony/Bundle/FrameworkBundle/Resources/config/debug_prod.xml

```diff
@@ -16,7 +16,7 @@
             <tag name="monolog.logger" channel="php" />
             <argument /><!-- Exception handler -->
             <argument type="service" id="logger" on-invalid="null" />
-            <argument /><!-- Log levels map for enabled error levels -->
+            <argument>null</argument><!-- Log levels map for enabled error levels -->
```

Comment:
For reference, here is the line that controls what happens in prod:
https://github.com/hhamon/symfony/blob/71e19de147f9d5c7688a827b2915510f298470eb/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php#L145